app/api/routes.py

```diff
@@ -144,5 +144,21 @@ def update_watering():
         'next_water': next_water
     }
     response = jsonify(results)
+    response.status_code = 201
+    return response
+
+@api.route('garden/plants/<int:id>')
+def get_gardenplant(id):
+    gardenplant = GardenPlant.query.get_or_404(id)
+    result = {
+        'id': gardenplant.id,
```

Comment:
Nice work on this route! I think changing this to `garden_plant_id` or `gardenplant_id` would be more explicit.

Reply:
Good call, just made that change.  See what ya think!